src/main/java/com/qualitestgroup/java/assessment/framework/pages/DemoPage.java

```diff
@@ -16,6 +16,18 @@
     public CommonPage page;
     public GlobalVars global;
 
+    @FindBy(xpath = "//div[@class='category-cards']/div")
```

Comment:
At the moment, there aren't other ways for us to fetch the elements apart from using xpath. There are no id's and using text is as fragile as xpath.

Reply:
I understand you point of view, but in my experience a page element is less likely to have its text change (unless it is dynamic) versus the page structure changing in some minor way due to other factors. Plus it is easier to maintain in case of future changes. In my opinion this would be a better xpath: "//div[@class='card-body']/h5[text()='Elements']"